packages/tools/client-debugger/client-debugger/src/Registry.ts

```diff
@@ -122,6 +125,41 @@ export class DebuggerRegistry extends TypedEventEmitter<DebuggerRegistryEvents>
 			this.postRegistryChange();
 			return true;
 		},
+		["CONNECT_CONTAINER"]: (untypedMessage) => {
+			const message = untypedMessage as ConnectContainerMessage;
```

Comment:
```suggestion
			const message: ConnectContainerMessage = untypedMessage as ConnectContainerMessage;
```

Reply:
Moved these three handlers to `FluidDebugger`. Removed undefine check and etc.